lib/bdev/bdev.c

```diff
@@ -857,6 +859,36 @@ spdk_bdev_reset(struct spdk_bdev *bdev, enum spdk_bdev_reset_type reset_type,
 	return rc;
 }
 
+struct spdk_bdev_io *
```

Comment:
I know NVMe admin passthru does not need an I/O channel, but for consistency with the other commands I'd prefer we require one in the interface. That unifies all of the code paths.

We just recently added an I/O channel to the reset interface for this same reason.